src/tools/html-checker/main.rs

```diff
@@ -0,0 +1,65 @@
+use std::env;
+use std::path::Path;
+use std::process::{Command, Output};
+
+fn check_html_file(file: &Path) -> bool {
+    let mut to_mute = vec!["PROPRIETARY_ATTRIBUTE"];
+    if file.file_name().and_then(|f| f.to_str()) == Some("settings.html") {
+        // In `settings.html`, we have empty `<span>` for the toggles.
+        to_mute.push("TRIM_EMPTY_ELEMENT");
+    }
+    let to_mute_s = to_mute.join(",");
+    let mut command = Command::new("tidy");
+    command
+        .arg("-e")
+        .arg("-q")
+        .arg("--mute-id") // this option is useful in case we want to mute more warnings
```

Comment:
For my reference:
```
`--mute-id`

This option indicates whether or not Tidy should display message ID's with    
each of its error reports. This could be useful if you wanted to use the      
mute configuration option in order to filter out certain report      
messages.   
```
Can you add a comment that you can look up options with `-help-option`? They don't show up in the default help.

Reply:
Read the man. :p But yes, I can add that.